### PR TITLE
Source Bing Ads: add fields to schema

### PR DESCRIPTION
**Pull request by LuliRib**
added field keyword to bing ads keyword_performance_report_daily

added field segments.device to google ads keyword_report

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json [hunk 1]
```diff
@@ -67,6 +67,9 @@
     "segments.date": {
       "type": ["null", "string"],
       "format": "date"
+    },
+     "segments.device": {
+      "type": ["null", "string"]
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json:72]: If I'm not wrong the field `segments.device` doesn't work with other `segments` fields.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json (hunk 1 above)
Comment (LuliRib) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json:72]: I thikk there is no problem. Attached doc
image
https://developers.google.com/google-ads/api/fields/v12/keyword_view?hl=en

**Comment by marcosmarxm** [comment]
@LuliRib can you bump the dockerfile version and documentation changelog too?

**Comment by LuliRib** [comment]
Hi, I see google ads test failed. Do u know why or what is the problem?


Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by LuliRib** [comment]
hi @marcosmarxm, how should we proceed? As bing ads changes pass, can it be pulled to airbyte? and can we repeat the google ads test?

**Comment by LuliRib** [comment]
Hi @marcosmarxm, first of all, I´m grateful for your help.
I saw bing-ads test passed.
I talked to google-ads api team and we figured out the problem with google ads. The segments.device is not compatible with 
"metrics.historical_quality_score". Can we just delete it from the schema?
If you can please help me a bit more with it, I need it urgently..
And again, thank you

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json (hunk 1 above)
Comment (LuliRib) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/schemas/keyword_report.json:69]: Found the problem and updated google_ads keyword_report.json file

**Comment by marcosmarxm** [comment]
@LuliRib can you split the contribution into 2 PRs one for Bing Ads and other for Google Ads? I'll ask the connector team to check your proposal of changes.

**Comment by LuliRib** [comment]
@marcosmarxm hi, not sure how to do it, keeping the data, tests of each source

**Review by marcosmarxm — APPROVED** [review]

**Comment by marcosmarxm** [comment]
@LuliRib please open a new branch/PR to make the changes to Google Ads.

**Comment by marcosmarxm** [comment]
@LuliRib because this connector is GA I'd requested to our connector team to make the final review before publishing the changes

**Comment by LuliRib** [comment]
Thank you so much!! Do you have an estimation on how much it would take?

And regarding the google ads pr. I´ll create a new one, but the tests that already ran wont appear.. and the conversation, ok?
________________________________

**Comment by LuliRib** [comment]
hi, just created a new pr
#20175 for google ads source
________________________________

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json [hunk 2]
```diff
@@ -14,6 +14,9 @@
     "KeywordId": {
       "type": ["null", "integer"]
     },
+    "Keyword": {
```
Comment (bazarnov) [inline airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json:17]: additionalProperties: true

Should be added to the changed schema, to support backward capability.

**Review by bazarnov — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json (hunk 2 above)
Comment (LuliRib) [reply airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json:17]: Hi @bazarnov, I just made the modification

Comment (bazarnov) [reply airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json:17]: Please add it to the root level of the schema, not to the property

Comment (LuliRib) [reply airbyte-integrations/connectors/source-bing-ads/source_bing_ads/schemas/keyword_performance_report.json:18]: Modified!

**Review by bazarnov — APPROVED** [review]

**Comment by LuliRib** [comment]
@bazarnov, thank you so much! What should I do next?

**Comment by bazarnov** [comment]
@marcosmarxm could you please assist here?

**Comment by LuliRib** [comment]
@bazarnov , @marcosmarxm I receive ONLY null values in the new keyword field. Is there a way you could check it?

**Comment by tomasx** [comment]
Same here Keyword field in keyword_performance_report_daily table always null

**Comment by LuliRib** [comment]
Hi @tomasx , I found the problem. Hopefully when it will be approved and merged to the code it will fix the problem

**Pull request merged**